articles/laravel-8-email-verification/index.md

```diff
@@ -46,24 +46,28 @@ cd verifyEmailApp
 php artisan serve
 ```
 
+`php artisan serve` command allows the application to be hosted locally.
+
 **Using Laravel Installer**  
-To build Laravel applications easily, Laravel provides a helpful command-line utility. Download the installer first: 
+There are numerous command-line utilities that make it easy to build applications using Laravel. We use the following command to download the Laravel installer: 
 
 ```bash
 composer global require laravel/installer
 ```
 
 To run the Laravel installer, you need to make sure that the Composer binary folder is within your `$PATH` variable.
-See if it is already in your `$PATH` variable, first by running the following command in your terminal:  
+Check if it is in your `$PATH` variable by running the following command in your terminal:  
 
 ```bash
 echo $PATH
 ```
-The performance should contain something like this if all is correct:  
+
+The correct output should look, as shown below:  
 `User/username/.vendor/composer/bin`  
-If not, change your `.bashrc` or, if you're using `ZSH`, your `.zshrc` so that it includes the path to the vendor directory for your composer.  
+
+Incase of any errors, change your `.bashrc` or, if you're using `ZSH`, your `.zshrc` so that it includes the path to the vendor directory for your composer.  
```

Comment:
When you run this command on the terminal, no error should be displayed, it's either there is an output or no output!

Reply:
you wrote 
```
The performance should contain something like this if all is correct:  
`User/username/.vendor/composer/bin`  
If not, change your `.bashrc` or, if you're using `ZSH`, your `.zshrc` so that it includes the path to the vendor directory for your composer. 
```
if not - means there is an error. Therefore `incase of any errors` seems appropriate.

Reply:
Something not showing an output while some result was expected means there is an error.

Reply:
> @jumamiller Already reviewed the suggested areas. I do not see any problem. Check my in-line comments on those specific issues above.

> @WanjaMike, I still believe this article is not up to the standard due to the minor changes you made! Programming has 'some' standard 'technical terms'  which I feel when 'misused', changes the entire meaning!

Reply:
@jumamiller Kindly, it's not up to standard due to the way you wrote it. The changes I have made across the article have improved its quality significantly.